test/box-luatest/gh_7974_force_recovery_bugs_test.lua

```diff
@@ -5,249 +5,247 @@ local t = require('luatest')
 local g = t.group()
 
 g.before_all(function()
-    t.skip('gh-8702')
+    t.tarantool.skip_if_not_debug()
```

Comment:
So now we can’t be sure force recovery works in release?

Reply:
https://github.com/tarantool/tarantool/issues/8723